src/app.tsx

```diff
@@ -25,10 +24,10 @@ const render = (Component: () => JSX.Element) => {
 render(Application);
 
 // Hot Module Replacement API
```

Comment:
Either resolve the error caused by this code or discuss if this can be removed with @roelvdboom. There shouldn't be any commented out code.